config/routes.rb

```diff
@@ -22,6 +22,8 @@
   get 'student_requests/getSpreadsheet' => 'student_requests#getSpreadsheet'
   get 'student_requests/uin/:uin' => 'student_requests#getStudentInformationByUin'
   
+  get 'student_requests/logout' => 'student_requests#logout'
```

Comment:
use post for logout 
http://stackoverflow.com/questions/3521290/logout-get-or-post